ui/redux/actions/settings.js

```diff
@@ -85,7 +98,18 @@ export function doClearDaemonSetting(key) {
 }
 // if doPopulate is applying settings, we don't want to cause a loop; doNotDispatch = true.
 export function doSetDaemonSetting(key, value, doNotDispatch = false) {
-  return dispatch => {
+  return (dispatch, getState) => {
+    const state = getState();
+    const ready = selectPrefsReady(state);
+
+    if (!ready) {
+      return dispatch(
+        doToast({
```

Comment:
This should be a separate function that can be re-used. 
```
if (!ready) {
  return dispatch(doAlertSyncNotReady())
}
```

(not sure about the name)

Reply:
doAlertWaitingForSync in actions/app.js